### PR TITLE
Yevgeny updated people

### PR DESCRIPTION
**Pull request by yevgenyr**
Updated my personal info in People db

### CODE REVIEW

**Comment by sbillinge** [comment]
Thanks so much for this @yevgenyr.  I will have to modify regolith to accept the linked_in ID in the schema, but I can do that quickly then it should pass CI....  It is a good addition.

File: db/people.yml [hunk 1]
```diff
@@ -4075,14 +4075,37 @@ yrakita:
   active: true
   aka:
     - Yevgeny Rakita
+    - Y. Rakita
+    - Yevgeny Rakita Shlafstein
   avatar: https://github.com/yevgenyr.png
-  bio: bio
+  bio: Yevgeny Rakita (Shlafstein) grew up in Israel, and after finishe his PhD,
```
Comment (sbillinge) [inline db/people.yml:4081]: typo

**Review by sbillinge — COMMENTED** [review]
one typo.  May as well catch it.... :)

**Comment by sbillinge** [comment]
Close and reopen to trigger CI after modifying regolith

**Comment by sbillinge** [comment]
It looks as if there is still an issue with the entry:
```
could not find expected ':'

  in "../db/people.yml", line 4083, column 3
```

**Comment by yevgenyr** [comment]
please try now. added double quotes and fixed typo. I left linkedin. Let me know if it is better to remove it for now and add when things are ready to host it

**Comment by sbillinge** [comment]
Thanks @yevgenyr I will merge like this.

I see from the deleted part that  the problem was just an indentation problem.  The text from line 2 onwards should have been indented a further two whitespaces.  No quotes etc. were actually needed, but I think it is fine as is (the html is building but I haven't looked how it renders).

**Pull request merged**